### PR TITLE
GitHub actions 2

### PR DESCRIPTION
**Pull request by rahulbcn27**
PR with ubuntu-22.04 added.

### CODE REVIEW

**Comment by rohityadavcloud** [comment]
@rahulbcn27 can you squash your commits?

File: .github/workflows/github-actions.yml [hunk 1]
```diff
@@ -25,7 +25,9 @@ concurrency:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+  
+    runs-on: ubuntu-22.04
```
Comment (rohityadavcloud) [inline .github/workflows/github-actions.yml:29]: Remove extra newline and could you rename the Github Actions job and yml file to something meaning ('Build' or build.yml may work)

Comment (rahulbcn27) [reply .github/workflows/github-actions.yml:28]: Done

**Comment by rahulbcn27** [comment]
> @rahulbcn27 can you squash your commits?

Learning how to it.

File: .github/workflows/github-actions.yml (hunk 1 above)
Comment (rahulbcn27) [reply .github/workflows/github-actions.yml:28]: Done

Comment (rahulbcn27) [reply .github/workflows/github-actions.yml:28]: Done

**Review by rohityadavcloud — APPROVED** [review]

**Pull request merged**